src/pages/Controllers/_controllers.scss

```diff
@@ -45,6 +45,10 @@
     }
   }
 
+  .disconnected-controller {
```

Comment:
Please use a state class syntax here. For example, `is-disconnected`. Makes the purpose of the class more clear to other developers. 

https://canonical-web-and-design.github.io/practices/coding/stylesheets.html#state-classes

Reply:
good call, done.